cli/lib/kontena/machine/cloud_config/cloudinit.yml

```diff
@@ -19,27 +19,28 @@ write_files:
       fs.inotify.max_user_instances = 8192
 coreos:
   units:
-    - name: 00-eth.network
-      runtime: true
+    - name: 10-weave.network
```

Comment:
I added this mainly to mask out any existing `10-weave.network` if upgrading the cloud-config for an existing node. If you have such a network unit, then you should remove it.